aunt-leahs-app/src/components/textField.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { TextField, Typography, makeStyles } from '@material-ui/core';
+import '../styles.css';
+
+const useStyles = makeStyles({
+    root:{
+        border: '2px solid #00A19B',
+        boxSizing: 'border-box',
+        fontFamily: 'Tahoma',
+        fontStyle: 'normal',
+        fontWeight: 'normal',
+        fontSize: '18px',
+        lineHeight: '22px',
+        padding: '0px 5px',
+    },
+});
```

Comment:
I think this should be moved to styles.css so that all of the component styling is in one place